lib/zinnia.js

```diff
@@ -91,6 +91,7 @@ const updateAllSourceFiles = async ({
             onFailedAttempt: err => {
               console.error(err)
               console.error(`Failed to download ${module} source. Retrying...`)
+              maybeReportErrorToSentry(err)
```

Comment:
Are you sure the original `err` contains information about the context, like the module name we were not able to download?

Alternative to consider:

```suggestion
              const msg = `Failed to download ${module} source. Retrying...`
              console.error(msg)
              maybeReportErrorToSentry(new Error(msg, { cause: err }))
```